### PR TITLE
rename tool commands

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: docs/cli-tool.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## Basic usage
+
+Install dependencies:
+
+```
+$ yarn
+```
+
+Run the cli tool:
```
Comment (escattone) [inline docs/cli-tool.md:11]: Nit:
```suggestion
Run the CLI tool:
```

File: docs/cli-tool.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+```
+$ yarn tool --help
+
+  cli.js 0.0.0
```
Comment (escattone) [inline docs/cli-tool.md:16]: Does `caporal` have a way to use a different name than the one from the command line? It'd be nice to use `tool` (or whatever else we decide) instead.

**Review by escattone — APPROVED** [review]
This looks great to me, thanks @fiji-flo. We can decide later if we want to change `tool` to something else.

File: docs/cli-tool.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  GLOBAL OPTIONS
+
+    -h, --help                           Display global help or command-related
+                                         help.
+    -V, --version                        Display version.
+    --no-color                           Disable use of colors in output.
+    -v, --verbose                        Verbose mode: will also output debug
+                                         messages.
+    --quiet                              Quiet mode - only displays warn and error
+                                         messages.
```
Comment (peterbe) [inline docs/cli-tool.md:45]: That's just bound to become out of date :)
Perhaps it's enough to say:

    Run the CLI tool:

    ```sh
    yarn tool --help
    ```

File: docs/cli-tool.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+### validate-redirect
```
Comment (peterbe) [inline docs/cli-tool.md:48]: Don't hate me for suggesting this but shouldn't be called `validate-redirects`?

File: docs/cli-tool.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+Move a document and its children. Adds the according redirects and stages
+changes in git (except for redirects).
+
+### edit
+
+Open a document by its slug in the preferred editor (as per the EDITOR
+environment variable).
+
+### crate
```
Comment (peterbe) [inline docs/cli-tool.md:77]:
```suggestion
### create
```

File: docs/cli-tool.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+Open a new document by its slug in the preferred editor (as per the EDITOR
+environment variable).
```
Comment (peterbe) [inline docs/cli-tool.md:80]: This is for the `edit` command.

File: docs/cli-tool.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+redirects).
+
+### move
+
+Move a document and its children. Adds the according redirects and stages
+changes in git (except for redirects).
+
+### edit
+
+Open a document by its slug in the preferred editor (as per the EDITOR
```
Comment (peterbe) [inline docs/cli-tool.md:74]:
```suggestion
Open a document by its slug in the preferred editor (as per the `EDITOR`
```

File: docs/cli-tool.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+Test whether an URL path is a redirect and display the according target.
+
+### add-redirect
+
+Add a redirect to the `_redirects.txt`.
+
+### delete
+
+Delete a document (and optionally all children) by its slug. Optionally add an
+redirect if deleting a single document. Also stages changes in git (except for
```
Comment (peterbe) [inline docs/cli-tool.md:64]:
```suggestion
redirect if deleting a single document. Also stages changes in `git` (except for
```

File: docs/cli-tool.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+Open a new document by its slug in the preferred editor (as per the EDITOR
+environment variable).
+
+### validate
+
+Run basic validation for a document (only verifies the slug for now).
+
+### preview
+
+Spawn a preview of a given slug in your browser. This depends on a running
```
Comment (peterbe) [inline docs/cli-tool.md:88]: I don't know what "spawn a preview" means. Perhaps it should have been called `open`.

**Review by peterbe — CHANGES_REQUESTED** [review]

File: docs/cli-tool.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
+# Yari's CLI Tool
+
+## Basic usage
+
+Install dependencies:
```
Comment (peterbe) [inline docs/cli-tool.md:5]: I don't think this belongs.

**Comment by fiji-flo** [comment]
I addressed all comments.

**Review by peterbe — APPROVED** [review]

**Pull request merged**